Headers/AppKit/GSSVGImageRep.h

```diff
@@ -0,0 +1,47 @@
+/* Definition of class GSSVGImageRep
```

Comment:
Should this file go into the AppKit directory or rather into our extension directory?

Reply:
This file has been removed.